src/config/index.ts

```diff
@@ -0,0 +1,7 @@
+export const database = {
```

Comment:
Could there perhaps be a way to use both the environment variables **or** a config file?

Reply:
Good afternoon David (Sorry for misspelling your name before, I just realized it now). You mean using one or the other, like aws (and co.) does? We could define a volume where a configuration file could be mounted on, then we would check if it exists before reading our configuration from environment. We could also define a special environment variable that would point to this file, then the user could mount it anywhere and set this variable (only the format would need to be respected). Is that it? Or am I still missing it.

Reply:
I will wait for you reply before doing anything here, since I am not sure as what to do. I think using volumes and environment variables are the most 'generic' ways of injecting any sort of configuration into  a container. Is there a third option? Can you give me a hint?